packages/lib/services/webhook.ts

```diff
@@ -8,11 +8,15 @@ import { ResourceNotFoundError, DatabaseError, InvalidInputError } from "@formbr
 
 export const getWebhooks = async (environmentId: string): Promise<TWebhook[]> => {
   try {
-    return await prisma.webhook.findMany({
+    const webhooks = await prisma.webhook.findMany({
       where: {
         environmentId: environmentId,
       },
     });
+    return webhooks.map((webhook) => ({
+      ...webhook,
+      source: webhook.source as "user" | "zapier" | null | undefined,
```

Comment:
if we change the prisma type we can also skip this `"user" | "zapier" | null | undefined,`